frontend/app/forms/reply-to-statement/reply-to-statement.pdf.component.js

```diff
@@ -0,0 +1,153 @@
+        <PositionedString dataKey="person.email" left={farLeft} top="28.16%" />
+
+        {/* Petitioner information */}
+        <PositionedCheckmark
+          dataKey="person.petitionerRepresentative"
+          left={petitionerCheckboxLeft}
+          top="31.50%"
+          shouldShow={data.person.petitionerRepresentative === "petitioner"}
+        />
+        <PositionedCheckmark
+          dataKey="person.petitionerRepresentative"
+          left={petitionerCheckboxLeft}
+          top={attorneyTop}
+          shouldShow={data.person.petitionerRepresentative === "attorney"}
+        />
+        <PositionedString
+          dataKey="person.petitionerBarNumber"
+          left="62.00%"
+          top={attorneyTop}
+        />
+
+        {/* Case information */}
+        <PositionedCheckmark
+          dataKey="case.courtType"
+          left="36.8%"
+          top={courtTypeTop}
+          shouldShow={data.case.courtType === "District"}
+        />
+        <PositionedCheckmark
+          dataKey="case.courtType"
+          left="48.2%"
+          top={courtTypeTop}
+          shouldShow={data.case.courtType === "Justice"}
+        />
+        <PositionedString
+          debugKey="courtDistrict"
+          left="25.9%"
+          top={courtDistrictTop}
+        >
+          {data.case.courtAddress &&
+            getJudicialDistrict(data.case.courtAddress, data.case.courtType)}
+        </PositionedString>
+        <PositionedString
+          debugKey="courtCounty"
+          left="52.00%"
+          top={courtDistrictTop}
+        >
+          {data.case.courtAddress &&
+            getCounty(data.case.courtAddress, data.case.courtType)}
+        </PositionedString>
+        <PositionedString
+          dataKey="case.courtAddress"
+          left="27.8%"
+          top="43.50%"
+        />
+
+        <PositionedString
+          debugKey="petitionerFullName"
+          left={farLeft}
+          top="54.5%"
+        >
+          {`${renderData("person.firstName")} ${renderData(
+            "person.middleName"
+          )} ${renderData("person.lastName")}`}
+        </PositionedString>
+        <PositionedString
+          dataKey="case.caseNumber"
+          left={caseInformationLeft}
+          top="54.5%"
+        />
+        <PositionedString
+          dataKey="case.judgeName"
+          left={caseInformationLeft}
+          top="59.5%"
+        />
+
+        <PositionedString
+          dataKey="case.opposingStatement"
+          left={disagreementLeft}
+          top="76.0%"
+          style={{
+            width: "71%",
+            overflowWrap: "anywhere",
+            lineHeight: "30px"
+          }}
+        />
+      </RenderPage>
+
+      <RenderPage url={secondPageUrl}>
+        <PositionedString
+          dataKey="case.disagreementReason"
+          left={disagreementLeft}
+          top="28.0%"
+          style={{
+            width: "71%",
+            overflowWrap: "anywhere",
+            lineHeight: "31px"
```

Comment:
Hmm which browsers have you tested this on? I'm a bit worried that the 8.5x11" print preview might display 31px differently in different browsers.

Reply:
I was developing in FireFox. I will check Chrome, Edge, and IE11 and get back to you on this.

Reply:
The app doesn't even load in IE11, so I don't have any pictures for it, but here's a comparison of Chrome, Edge, and FireFox:
![image](https://user-images.githubusercontent.com/8525915/66720729-e4847a80-edbd-11e9-968e-2644ee66a2fd.png)

Unfortunately, having a bunch of really long words causes more frequent line breaks, so there's an overflow of text from the first page area. But the text does align properly with 31px as a lineHeight in all browsers.

I don't think many of our users will be rolling their hand over their keyboard like I did just to get a long input, but we might need to account for unexpected line breaks, especially if the user presses "enter" a lot.

Reply:
👍 thanks for doing that. I think that we don't need to worry about overflowing beyond the allotted area. And am happy to see that it works in all browsers.

Reply:
(except IE which we don't support)

Reply:
Are we good to resolve this conversation, then?